c10/util/Logging.cpp

```diff
@@ -254,6 +273,9 @@ C10_DEFINE_int(
     "The minimum log level that caffe2 will output.");
 
 namespace c10 {
+
+void initLogging() {}
```

Comment:
Nit: Add a comment explaining how logging will be done when GLOG is not available? i.e. I'm guessing the logging APIs will be from `#include <c10/util/logging_is_not_google_glog.h>` which is included above, but might help future developers understand